cvat/apps/engine/cron.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import importlib
+import logging
+import os
+import os.path as osp
+from contextlib import suppress
+from datetime import timedelta
+from pathlib import Path
+
+from django.db.models import QuerySet
+from django.utils import timezone
+
+from cvat.apps.dataset_manager.util import (
+    ExportCacheManager,
+    LockNotAvailableError,
+    get_export_cache_lock,
+)
+from cvat.apps.dataset_manager.views import (
+    EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT,
+    EXPORT_CACHE_LOCK_TTL,
+    get_export_cache_ttl,
+    log_exception,
+)
+from cvat.apps.engine.log import ServerLogManager
+from cvat.apps.engine.models import Job, Project, Task
+
+
+class FileIsBeingUsedError(Exception):
+    pass
+
+
+def clear_export_cache(file_path: str, logger: logging.Logger) -> None:
+    try:
+        with get_export_cache_lock(
+            file_path,
+            block=True,
+            acquire_timeout=EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT,
+            ttl=EXPORT_CACHE_LOCK_TTL,
+        ):
+            if not osp.exists(file_path):
+                raise FileNotFoundError(f"Export cache file {file_path} doesn't exist")
```

Comment:
This seems like a redundant check. If the file does not exist, then `osp.getmtime` will fail with the same exception.